tests/flow/test_index_delete.py

```diff
@@ -1,145 +1,301 @@
 
-        # validate no indexes in graph
-        result = list_indicies(g)
-        self.env.assertEquals(len(result.result_set), 0)
+        # make sure 'NewAttr' is not part of the graph
+        result = self.g.query("CALL db.propertyKeys()")
+        self.env.assertFalse('NewAttr' in result.result_set[0])
+
+        # drop index over 'L', 'age'
+        result = drop_node_range_index(self.g, 'L', 'age')
 
-    def test05_drop_index_during_population(self):
+    def test_07_drop_index_during_population(self):
         # 1. populate a graph
         # 2. create an index and wait for it to be sync
         # 3. constantly update indexed entities
         # 4. drop index
         # 5. validate execution-plan + indexes report
 
+        #-----------------------------------------------------------------------
         # populate a graph
+        #-----------------------------------------------------------------------
         q = "UNWIND range(0, 1000) AS x CREATE (:N{v:x})"
-        g.query(q)
+        self.g.query(q)
 
+        #-----------------------------------------------------------------------
         # create an index and wait for it to be sync
-        create_node_exact_match_index(g, 'N', 'v', sync=True)
+        #-----------------------------------------------------------------------
+        create_node_range_index(self.g, 'N', 'v', sync=True)
 
+        #-----------------------------------------------------------------------
         # constantly update indexed entities
         # drop index
         # validate execution-plan + indexes report
+        #-----------------------------------------------------------------------
         start_idx = 0
         end_idx = 100
         for i in range(start_idx, end_idx):
             # update indexed entities
             q = f"MATCH (n:N) WHERE n.v = {i} SET n.v = -n.v"
-            g.query(q)
+            self.g.query(q)
 
             if i < end_idx / 2:
                 # validate execution-plan + indexes report
-                plan = g.execution_plan(q)
+                plan = self.g.execution_plan(q)
                 self.env.assertIn("Index", plan)
 
-                indicies = list_indicies(g).result_set
+                indicies = list_indicies(self.g).result_set
                 self.env.assertEquals(len(indicies), 1)
 
             elif i == end_idx / 2:
                 # drop index
-                drop_exact_match_index(g, 'N', 'v')
+                drop_node_range_index(self.g, 'N', 'v')
 
             else:
                 # validate execution-plan + indexes report
-                plan = g.execution_plan(q)
+                plan = self.g.execution_plan(q)
                 self.env.assertNotIn("Index", plan)
 
-                indicies = list_indicies(g).result_set
+                indicies = list_indicies(self.g).result_set
                 self.env.assertEquals(len(indicies), 0)
 
-    def test06_reset_order(self):
+    def test_08_reset_order(self):
```

Comment:
move this test

Reply:
Couldn't find an alternative test file for this test, the only reason for this test to remain here is related to the fact that it relies on the existence of an index.